modules/rosa-sdpolicy-platform.adoc

```diff
@@ -0,0 +1,86 @@
+:system-module-type: CONCEPT
+// Module included in the following assemblies:
+//
+// * assemblies/rosa-service-definition.adoc
+
+[id="rosa-sdpolicy-platform_{context}"]
+= Platform
+
+== Cluster backup policy
+
+[IMPORTANT]
+====
+It is critical that customers have a backup plan for their applications and application data.
+====
+Application and application data backups are not a part of the {product-title} service.
+All Kubernetes objects and PVs in each {product-title} cluster are backed up to facilitate a prompt recovery in the unlikely event that a cluster becomes irreparably inoperable.
+
+The backups are stored in a secure object storage (Multi Availability Zone) bucket in the same account as the cluster.
+Node root volumes are not backed up as Red Hat Enterprise Linux CoreOS is fully managed by the OpenShift Container Platform cluster and no stateful data should be stored on a node's root volume.
+
+The following table shows the frequency of backups:
+[cols="4",options="header"]
+|===
+
+|Component
+|Snapshot Frequency
+|Retention
+|Notes
+
+|Full object store backup, all cluster PVs
+|Daily at 0100 UTC
+|7 days
+|This is a full backup of all kubernetes objects, as well as all mounted PVs in the cluster.
+
+|Full object store backup, all cluster PVs
+|Weekly on Mondays at 0200 UTC
+|30 days
+|This is a full backup of all kubernetes objects, as well as all mounted PVs in the cluster.
+
+|Full object store backup
+|Hourly at 17 minutes past the hour
+|24 hours
+|This is a full backup of all kubernetes objects. No PVs are backed up in this backup schedule.
+
+|===
+
+== Autoscaling
+Node autoscaling is available on {product-title}.
+
+== Daemonsets
+Customers may create and run DaemonSets on {product-title}. In order to restrict DaemonSets to only running on worker nodes, use the following nodeSelector:
+----
+...
+spec:
+  nodeSelector:
+    role: worker
+...
+----
+
+== Multiple availability zone
+In a multiple availability zone cluster, master nodes are distributed across availability zones and at least one worker nodes are required in each availability zone.
+
+== Node labels
+Custom node labels are created by Red Hat during node creation and cannot be changed on {product-title} clusters at this time.
+
+== Openshift version
+{product-title} is run as a service and is kept up-to-date with the latest OpenShift Container Platform version. Upgrade scheduling to the latest version is available.
+
+== Upgrades
+Upgrades can be scheduled using the `rosa` CLI utility.
+
+Patch level (also known as z-stream; x.y.Z) updates are applied automatically the week following their release as long as {product-title}-specific end-to-end tests pass.
+
+Minor version updates (x.Y.z) may include Kubernetes version upgrades and/or API changes. Therefore, customers are notified by email two weeks in advance before these upgrades are automatically applied.
+
+== Window containers
```

Comment:
Should this be "Windows containers" or "Windows Containers"?

Reply:
Windows Containers